### PR TITLE
Pivot

### PR DESCRIPTION
**Pull request by piazzaana**

### CODE REVIEW

File: docs/milestones.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Project Milestones
```
Comment (markmichon) [inline docs/milestones.md]: These project milestones need to be filled in. Links should be provided for the initial pitch video (if available) and the initial spec. See [the milestones section of the first assignment](https://github.com/reactivepixel/DWA/blob/master/ClassContent/Module-2-Pivot/Milestone-1.md#project-milestones) for further details.

File: docs/readme.md [hunk 2]
```diff
@@ -1 +1,124 @@
-readme.file
+# Chicken & Fries
```
Comment (markmichon) [inline docs/readme.md]: Break the installation instructions out of this readme and into a readme that lives at the root of the repo. The readme in the docs folder starts where the table of contents begins.

File: docs/readme.md [hunk 3]
```diff
@@ -1 +1,124 @@
...
+* [Project Milestones]()
+    * [Initial Pitch]()
+    * [Initial Project Spec]()
+    * [MVP]()
+    * [Pivot]()
+    * [Alpha Release]()
+    * [Beta Release]()
+    * [Public release]()
+
+#Brief
```
Comment (markmichon) [inline docs/readme.md]: Currently some of this content is duplicated or split between this main readme file and more specific sub-files. You have two ways you can approach this:
1. Keep all the specification information in this file like it is now (with requested changes) and then remove the extra files (brief.md, milestones.md, etc).
2. Use this readme file as just a table of contents, with links to those extra files.

File: docs/readme.md [hunk 4]
```diff
@@ -1 +1,124 @@
...
+Open your terminal window and run the following command
+```
+nodemon bin/www
+```
+Open your favorite browser and go to
+```
+localhost:3000
+```
+
+#Table Of Contents
```
Comment (markmichon) [inline docs/readme.md]: You'll want to fill in these links in the table of contents to lead to the sections of your project spec.

File: docs/readme.md [hunk 5]
```diff
@@ -1 +1,124 @@
...
+------------ | -------------
+Database | MongoDB
+Web Server | Express
+View Engine | Pug
+Version Control | Git ([GitHub](https://github.com/piazzaana/chicken-fries))
+###Programming Languages
+* NodeJS v10.5.0
+* MongoDB
+###Integrations
+###Deployment Workflow
```
Comment (markmichon) [inline docs/readme.md]: It's time to start thinking about your deployment strategy. See Research I in FSO for some potential options. We'd suggest heroku as a good choice, but you're free to explore some of the other options listed. You don't have to set deployment up for the pivot milestone, but you need to decide which you plan to use.

File: docs/brief.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+#Brief
+On this brief I will go through the audience, the problem, the solution, my unique value proposition, and pitch.
+I'll identify the audience and the problem, will come up with a solution using my unique value proposition, and will deliver a pitch.
```
[the remark this answers is not in the document]
Comment (piazzaana) [reply docs/brief.md]: Fixed it

File: docs/readme.md (hunk 2 above)
Comment (piazzaana) [reply docs/readme.md]: fixed it

File: docs/readme.md (hunk 3 above)
Comment (piazzaana) [reply docs/readme.md]: fixed it

File: docs/readme.md (hunk 5 above)
Comment (piazzaana) [reply docs/readme.md]: Working on it

Comment (piazzaana) [reply docs/readme.md]: @markmichon I decide to go with Heroku

**Comment by markmichon** [comment]
@piazzaana The updates look good 👍. You just need to resolve that merge conflict that I suspect is caused by moving files around. Once that is done you are approved to proceed.

**Comment by piazzaana** [comment]
@markmichon I did YAY!  :)

**Comment by piazzaana** [comment]
Should I delete this branch?

**Pull request merged**